corehq/apps/hqwebapp/static/hqwebapp/js/components/pagination.js

```diff
@@ -41,12 +41,15 @@ hqDefine('hqwebapp/js/components/pagination', [
             if (params.limitSlug) {
                 self.initialValues.limit = parseInt(self._url.searchParams.get(params.limitSlug));
             }
+            self.watchChanges = (typeof params.watchChanges !== 'undefined') ? params.watchChanges : true;
```

Comment:
This file has parameter docs at the top - could you add a description of `watchChanges` there?